pkgs/by-name/so/sointu/package.nix

```
@@ -0,0 +1,56 @@
+{
+  buildGo123Module,
+  fetchFromGitHub,
+  lib,
+  pkg-config,
+  alsa-lib,
+  libGL,
+  libxkbcommon,
+  vulkan-headers,
+  wayland,
+  xorg,
+}:
+
+buildGo123Module {
+  pname = "sointu";
+  version = "0.4.1-unstable-2025-08-13";
+
+  src = fetchFromGitHub {
+    owner = "vsariola";
+    repo = "sointu";
+    rev = "74fea4138fd788eddeb726440c872937de56fd1c";
+    sha256 = "sha256-kHK35Bt/+ucPCsFE3p72J3jSHzhOK9QKtJPG+3grBvs=";
+  };
+
+  nativeBuildInputs = [
+    pkg-config
+  ];
+  buildInputs = [
+    alsa-lib
+    libGL
+    libxkbcommon
+    vulkan-headers
+    wayland
+    xorg.libX11
+    xorg.libXcursor
+    xorg.libXfixes
+  ];
+
+  proxyVendor = true;
+  vendorHash = "sha256-gLDLKqu6k7/nwv6xHUE6MIYrbQFfVFAuUiMbLptcE5k=";
+
+  subPackages = [
+    "cmd/sointu-track"
+    "cmd/sointu-compile"
+    "cmd/sointu-play"
+  ];
+
+  meta = with lib; {
+    description = "Fork of 4klang that can target 386, amd64 and WebAssembly. Tools run on Windows, Mac & Linux";
```

```suggestion
    description = "Fork of 4klang that can target 386, amd64 and WebAssembly";
```

Supported platforms are implied by `meta.platforms`.